.circleci/config.yml

```diff
@@ -7,54 +7,41 @@ jobs:
     steps:
       - checkout
       - run: ./tests/newversioncheck.sh
-  emerge all ebuilds:
-    machine: true
-    resource_class: large
+
+  pkgcheck all:
+    docker:
+      - image: registry.gitlab.fem-net.de/gentoo/fem-overlay-ci-image:master
```

Comment:
Not sure if we/you should depend on a (for me random/unvetted) third-party Docker image.
You can just use the upstream gentoo image with a volume container with the portage image to get the portage tree.

Reply:
Well, that image is built with a GitLab CI pipeline in a [public repository](https://gitlab.fem-net.de/gentoo/fem-overlay-ci-image/). This is way more trustworthy than all the images on Docker Hub which don't link to their repo and where you can't be sure the image published is actually built from the Dockerfile linked in the README. And yet people blindly accept these kinds of Docker images as well (not saying you do – you obviously used the official images which can be considered trusted).
Also, you can have a look at the commit history and you'll see that the Dockerfile in this repo is written by me (since I didn't find any image on Docker Hub which satisfied my requirements).

The solution you proposed still has the issue of requiring to install the required tools during each CI run. This will cause problems after version bumps of important Gentoo packages like Perl which require rebuilding a bunch of stuff on the system, since it will blow up CI run times and maybe even timeout the jobs. That stuff is one of the reasons I do all the work during the build of the Docker image.

I suppose I could switch back to the old approach. This will, however, introduce the aforementioned problems without bringing (at least in my opinion) any benefit, since I consider the trust-issue with the fem-overlay-ci-image non-existent, given that the entire build process can be seen and monitored publicly.